spec/cassette/authentication/user_spec.rb

```diff
@@ -8,63 +8,93 @@
   describe '#initialize' do
     context 'without a config' do
       it 'forwards authorities parsing' do
-        expect(Cassette::Authentication::Authorities).to receive(:new).with('[CUSTOMERAPI, SAPI]', nil)
-        Cassette::Authentication::User.new(login: 'john.doe', name: 'John Doe', authorities: '[CUSTOMERAPI, SAPI]')
+        allow(Cassette::Authentication::Authorities).to receive(:new).with('[CUSTOMERAPI, SAPI]', nil)
```

Comment:
After this change from `expect` to `allow`, is this spec still testing the original behavior? What do you think about using `expect(...).to have_received` along with this `allow`, instead? This way you still ensure the original expectancy when the test used `expect(...).to receive`, but without the rubocop warning :)